Dockerfile

```diff
@@ -1,4 +1,6 @@
 FROM ubuntu:20.04 as build-image
+ARG TARGETARCH
+ENV TARGETARCH "${TARGETARCH}"
```

Comment:
Wondering if we could simplify this by having `ARG` have a default value

```
ARG TARGETARCH=amd64
```

And that way avoid setting it in the follow up commands?